packages/apollo-federation/src/service/__tests__/buildFederatedSchema.test.ts

```diff
@@ -164,6 +165,7 @@ type Query {
 
       expect(schema.getQueryType()).toMatchInlineSnapshot(`
 type QueryRoot {
+  """"""
```

Comment:
Similarly to my previous comment: This inclusion of a blank comment on the `Query` type in this snapshot from `getQueryType` is a surprising change worth investigating.